From 818e15336b395b5c90729ffd81530bd5bbb5c073 Mon Sep 17 00:00:00 2001
From: Steven Chamberlain <steven@pyro.eu.org>
Date: Thu, 17 Dec 2015 02:15:58 +0000
Subject: =?UTF-8?q?kFreeBSD=2010=20(possibly=20only=20with=20softupdates?=
 =?UTF-8?q?=20enabled)=20may=20defer=0Acalculating=20the=20mtime=20for=20m?=
 =?UTF-8?q?ore=20than=202=20seconds.=20=20Work=20around=20this=0Awith=20a?=
 =?UTF-8?q?=20stat()=20call=20to=20calculate=20the=20mtime=20immediately.?=

(Modified to only stat() on kfreebsd by Niko Tyni and rebased against
Debian wheezy by Dominic Hargreaves)

Bug-Debian: https://bugs.debian.org/796798
Patch-Name: debian/kfreebsd-softupdates.diff
---
 t/op/stat.t | 1 +
 1 file changed, 1 insertion(+)

diff --git a/t/op/stat.t b/t/op/stat.t
index 1808ee4..7688725 100644
--- a/t/op/stat.t
+++ b/t/op/stat.t
@@ -90,6 +90,7 @@ sleep 3 if $funky_FAT_timestamps;
 print FOO "Now is the time for all good men to come to.\n";
 close(FOO);
 
+stat($tmpfile) if $^O =~ /^gnukfreebsd/;       # Work around Debian Bug#796798
 sleep 2;
 
 
